scripts/jest/environment.js

```diff
@@ -1 +1,3 @@
+require('core-js/modules/es6.object.assign');
```

Comment:
@zpao The `core-js/library` modules avoid global namespace pollution. I found `core-js/modules` from snooping around, but now I think I might be wrong. Let me rtfm and get back to you.